### PR TITLE
Ci misc

### PR DESCRIPTION
**Pull request by heavenlyhash**
Some small tweaks to CI initialization.

### CODE REVIEW

File: test/cluster/cluster.go [hunk 1]
```diff
@@ -109,21 +109,21 @@ func (c *Cluster) BuildFlynn(rootFS, commit string, merge bool) (string, error)
 		},
 	})
 	if err != nil {
-		return "", err
+		return build.Drive("hda").FS, err
 	}
 	c.log("Booting build instance...")
 	if err := build.Start(); err != nil {
-		return "", fmt.Errorf("error starting build instance: %s", err)
+		return build.Drive("hda").FS fmt.Errorf("error starting build instance: %s", err)
```
Comment (titanous) [inline test/cluster/cluster.go]: Missing comma?

File: test/main.go [hunk 2]
```diff
@@ -78,10 +78,12 @@ func main() {
 		if err != nil {
 			testCluster.Shutdown()
 			log.Println("could not build flynn: ", err)
+			os.RemoveAll(rootFS)
 			return
 		}
 		if args.KeepRootFS {
 			fmt.Println("Built Flynn in rootfs:", rootFS)
+			os.Exit(0)
```
Comment (titanous) [inline test/main.go]: The current flag name/definition does not imply this:

```text
-keep-rootfs don't remove the rootfs which was built to run the tests
```

Change the flag if this behavior makes sense.

File: test/main.go [hunk 3]
```diff
@@ -78,10 +78,12 @@ func main() {
 		if err != nil {
 			testCluster.Shutdown()
 			log.Println("could not build flynn: ", err)
+			os.RemoveAll(rootFS)
```
Comment (titanous) [inline test/main.go]: Should probably wrap this in a `if rootFS != ""`

**Comment by titanous** [comment]
LGTM

**Comment by lmars** [comment]
LGTM

**Pull request merged**